### PR TITLE
In the AR, add a label with the name of the flower or nest you're currently looking at

### LINKED ISSUE #153: Move the flower/nest name somewhere other than the interact button
We'd probably rather have an icon on the interact button, if we don't get rid of it entirely and replace it with some sort of gesture. In any case, we should probably find a new home for the flower/nest's name.

### PR DESCRIPTION
**Pull request by helloworld12321**
Using the layout and styling we discussed on the Tuesday Nov. 16th meeting :slightly_smiling_face: 

Closes #153.

### CODE REVIEW

**Comment by helloworld12321** [comment]
Screen Shot 2020-11-21 at 13 23 43
Screen Shot 2020-11-21 at 13 20 22
Screen Shot 2020-11-21 at 13 18 29
Screen Shot 2020-11-21 at 13 19 38

File: src/_variables.scss [hunk 1]
```diff
@@ -20,3 +20,9 @@ $background: map-get($theme, background);
 // bars and our bottom bars.
 $menu-bar-height: 64px;
 $menu-bar-padding: 16px;
+
+// Some breakpoints for media queries.
+$tablet-portrait-min-width: 600px;
+$tablet-landscape-min-width: 900px;
```
Comment (floogulinc) [inline src/_variables.scss:26]: Suggestion: use the official Material design breakpoints: https://material.io/design/layout/responsive-layout-grid.html#breakpoints

This code may be useful (the strings are all css `@media` queries):

https://github.com/angular/components/blob/bb2bb9c3065ec1a7a845de072305893a7378383a/src/cdk/layout/breakpoints.ts#L10-L31

Comment (helloworld12321) [reply src/_variables.scss:26]: Ah, sweet, thank you! Those material breakpoints are super useful. Will do!

**Review by kklamberty — APPROVED** [review]
I think this looks great! It worked for me on my machine. I used Firefox, and it did ask me for permission to use the camera each time I was in a new round.

**Comment by helloworld12321** [comment]
Okay, here it is with titlecase labels! :slightly_smiling_face:

iPhone 5/SE:
Screen Shot 2020-11-24 at 14 34 58

iPad Mini:
Screen Shot 2020-11-24 at 14 34 14

Nexus 10:
Screen Shot 2020-11-24 at 14 32 01

iPhone X/XS:
Screen Shot 2020-11-24 at 14 32 55

**Review by floogulinc — APPROVED** [review]

**Review by kklamberty — APPROVED** [review]

**Comment by helloworld12321** [comment]
Thanks very much—merging! :)

**Pull request merged**